.github/workflows/build.yml

```diff
@@ -43,6 +43,10 @@ jobs:
           - stage3-ppc64le-musl-hardened-openrc
           - stage3-ppc64le-openrc
           - stage3-ppc64le-systemd
+          - stage3-rv64_lp64-openrc
+          - stage3-rv64_lp64-systemd
+          - stage3-rv64_lp64d-openrc
+          - stage3-rv64_lp64d-systemd
```

Comment:
What about the `rv64_multilib` variants?

Reply:
I think Dilfridge is still testing and experimenting multi-lib support. It is not marked stable in the profile list yet.

Reply:
In that case, we could probably add a note in the relevant README section, explaining why these variants are not built (e.g., marked as experimental or something).

Reply:
sure, added in new commits